tests/sles4sap/saptune/mr_test.pm

```diff
@@ -300,7 +298,7 @@ sub run {
     } elsif ($test =~ m/^(x86_64|ppc64le)$/) {
         $self->test_x86_64  if (check_var('BACKEND', 'ipmi'));
         $self->test_ppc64le if (get_var('OFW'));
-        $self->test_bsc1152598;
+        #$self->test_bsc1152598;
```

Comment:
Isn't there a condition from the system such as saptune's version number or something similar that can be used to decided whether `$self->test_bsc1152598` gets called? Commenting a function call introduced on #9321 (just 6 PRs before this one), only to introduce eventually another PR just to remove the comment on the line seems overkill IMO. Makes me think that #9321 shouldn't have been merged until we had the assurance that it was needed.

Reply:
Yes, the maintenance update was just rejected as Angela wants to add another fix to the package.